modules/ossm-exposing-a-service-by-using-the-kubernetes-gateway-api.adoc

```diff
@@ -5,15 +5,16 @@
 [id="ossm-exposing-a-service-by-using-the-kubernetes-gateway-api_{context}"]
 = Exposing a service by using the {k8s} Gateway API 
```

Comment:
Shall we name this as follows? Because, I see we have a separate link for ".... in ambient mode"

"Exposing a service by using the {k8s} Gateway API in sidecar mode"

CC: @cam-garrison, Jamie

Reply:
Yes, agreed. Would be best to specify this is for sidecar mode.